src/containers/ConfirmationValidatorApi/ConfirmationValidatorApiBanks/index.tsx

```diff
@@ -9,19 +9,51 @@ const ConfirmationValidatorApiBanks: FC = () => {
 
       <DocEndpoint endpoint="/banks" method="GET" />
       <RequestResponseSnippet
-        code={`[
-  {
-    "account_number": "5e12967707909e62b2bb2036c209085a784fabbc3deccefee70052b6181c8ed8",
-    "ip_address": "192.168.1.232",
-    "node_identifier": "d5356888dc9303e44ce52b1e06c3165a7759b9df1e6a6dfbd33ee1c3df1ab4d1",
-    "port": 8000,
+        code={`{
```

Comment:
For the Listing requests, we have decided to add the table of query params kindly add that.
[here](https://github.com/thenewboston-developers/Website/pull/982/files#diff-c4eee41d3a8c9ab0d7ec0cb4717b0991755b71e565eb72fe46e6217c8b5be168) is the reference of how that should look like

Reply:
also for the response, ideal response would be this one (the response given by the url below since it has the next object as well.)
http://54.183.17.224/banks?limit=2

Reply:
Okay, got it.